src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Media/ImageBrushTests/BorderImageBrush.xaml.cs

```diff
@@ -16,7 +16,7 @@
 
 namespace Uno.UI.Samples.UITests.ImageBrushTestControl
 {
-	[SampleControlInfo("ImageBrushTestControl", "BorderImageBrush")]
+	[SampleControlInfo("ImageBrush", "BorderImageBrush")]
```

Comment:
You can't validate for visual breaking changes if you change the category name.

Reply:
Right, I can undo this rename, and change it later in a separated commit so we will be sure to not have any bug in it.